FrontEnd/Starfire.cs

```diff
@@ -59,7 +59,7 @@ public Starfire()
             this.TopMost = true;
             this.WindowState = FormWindowState.Normal;
             Starfire_Status.Text = "STATUS: N/A";
-            SubscribeToKeyPressesForHotkeys();
+            GlobalHookKeyDown();
```

Comment:
This isn't really necessary as it will be called once every frame in the background thread